src/main/java/eventsg/backend/controller/UserController.java

```diff
@@ -1,4 +1,80 @@
 package eventsg.backend.controller;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+import eventsg.backend.model.User;
+import eventsg.backend.service.UserServiceImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RequestMapping("api/v1/user")
+@RestController
 public class UserController {
+
+    private final UserServiceImpl userService;
```

Comment:
doudou says we need to pass in the UserService interface here, then implement the UserService interface with UserServiceImpl